design_practice/src/components/Subscription/MainSubscription/MainSubscription.js

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import movie from "../../../assets/images/movie.jpg"
+
+import "./MainSubscription.sass"
+
+const MainSubscription = () => {
+  return (
+    <section className="subscription">
+      <div className="subscription__section">
+        <img
+          className="subscription__image"
+          alt="should show movie"
+          src={movie}
+        />
+      </div>
+      <div className="subscription__section">
+        <div className="subscription__title bold-text">
+          Communicate.
+          <br />
+          Collaborate. Complete.
+        </div>
+        <div className="subscription__subtitle">
+          The remote collaboration tool to help you communicate effectively &
+          work productively
+        </div>
+        <div>
+          <label className="form-label">Email address</label>
+          <input className="form-input" type="email" />
+          <div className="subscription__actions">
+            <div>
+              <div>
```

Comment:
`div` soup, you use it just to have text in a new line, so you could use `<br />` instead